File: Modules/Common/LES/SIE.Web.LES/StockOrders/Commands/StockOrderMergeIssuedAddCommand.js
SIE.defineCommand('SIE.Web.LES.StockOrders.Commands.StockOrderMergeIssuedAddCommand', {
    extend: 'SIE.cmd.Add',
    meta: { text: "添加", group: "edit", iconCls: "icon-AddEntity icon-green" },
    onItemCreated: function (entity) {
        entity.setState(0);
    },
});

File: Modules/Common/LES/SIE.Web.LES/StockOrders/Commands/StockOrderMergeIssuedAddCommand.cs
using SIE.Web.Command;

namespace SIE.Web.LES.StockOrders.Commands
{
    public class StockOrderMergeIssuedAddCommand : ViewCommand
    {
        protected override object Excute(ViewArgs args, string scope)
        {
            return true;
        }
    }
}
